frontstage/templates/sign-in/sign-in.account-locked.html

```diff
@@ -0,0 +1,37 @@
+{% import 'partials/section.html' as section %}
+{% extends "layouts/_twocol.html" %}
+
+{% block page_title %}Trouble signing in? - ONS Business Surveys{% endblock %}
+
+{% block main %}
+    <h1 class="saturn">Check your email</h1>
+
+    <p>You've tried to sign in few times with the wrong details.</p>
+    <p>If your email is registered to an account, we will send an email explaining how to sign in.</p>
```

Comment:
This based of the content should be passing in your actual email address the user tried registering with

Reply:
no we dont want to pass email about, plus this is an anonymous page, we dont want to let the people know which email it is

Reply:
ill double check with someone